fenix/app/src/main/java/org/mozilla/fenix/home/topsites/TopSites.kt

```diff
@@ -312,6 +314,14 @@ private fun TopSiteItem(
             }
         }
     }
+
+    val context = LocalContext.current
+    val activity = context as? HomeActivity
+    LaunchedEffect(Unit) {
+        activity?.let {
+            StartupTimeline.onTopSitesItemBound(it)
+        }
```

Comment:
drive by: this will very likely break previews, and obtaining the activity reference in Compose code isn't ideal. It might make more sense to pass in a lambda to handle this instead.

Reply:
Agreed, @DreVla let's move this into lambda that is passed into our @Composables from the fragment. We can call it `onTopSitesItemBound`